scripts/coverage_scan.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo
+
+pull_number=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+export PULL_REQUEST_KEY=$pull_number
+
+echo "running lazy sonar tests."
+
+export OUTPUTDIR="coverage"
+mkdir -p "$OUTPUTDIR"
+
+#SONAR_ORGANIZATION="$SONAR_ORG"
```

Comment:
Remove this commented line if SONAR_ORGANIZATION is not needed.

Reply:
yep